### ISSUE TITLE
Outdated version in cargo

### ISSUE THREAD

**Issue by zyrgo** [dwisiswant0/ppfuzz#48]
Only ppfuzz 1.0.1 is available in cargo, while latest version being 1.0.2

**Comment by edoardottt** [comment]
Yep, v1.0.2 should be uploaded to cargo, v1.0.1 is the broken one...
@dwisiswant0

**Comment by zyrgo** [comment]
![image](https://github.com/dwisiswant0/ppfuzz/assets/45917995/cff88ba0-6baf-44e4-8720-50d25c8f5007)
@edoardottt still while running `cargo install ppfuzz` above error shows, its still 1.0.1.
